iac-extensions/docker/src/main/java/org/sonar/iac/docker/plugin/DockerSensor.java

```diff
@@ -80,13 +80,21 @@ public void describe(SensorDescriptor descriptor) {
   protected FilePredicate mainFilePredicate(SensorContext sensorContext) {
     FileSystem fileSystem = sensorContext.fileSystem();
     FilePredicates p = fileSystem.predicates();
-    return p.and(
-      p.or(
+
+    FilePredicate pathPatterns = p.matchesPathPattern("**/Dockerfile.*");
+    if (!isNotSonarLintContext(sensorContext)) {
```

Comment:
Does this condition need a TODO to be removed after SonarLint adopts new API too?

Reply:
Added a reference to https://sonarsource.atlassian.net/browse/SLCORE-526